core-java-modules/core-java-nio/pom.xml

```diff
@@ -0,0 +1,16 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Let's indent the file using spaces rather than tabs.

Reply:
fixed.